src/annotator/components/Buckets.js

```diff
@@ -55,18 +51,20 @@ function BucketButton({ bucket, onFocusAnnotations, onSelectAnnotations }) {
  *
  * @param {object} props
  *   @param {Bucket} props.bucket
- *   @param {'up'|'down'} props.direction
- *   @param {(a: Anchor) => void} props.scrollToAnchor - Callback invoked to
- *     scroll the document to a given anchor
+ *   @param {'down'|'up'} props.direction
+ *   @param {(tags: string[], direction: 'down'|'up') => void} props.onScrollToClosestOffScreenAnchor
+ *     - Callback invoked to scroll the document to a given anchor
```

Comment:
The comment refers to _anchor_ singular, but there are multiple tags. IMO the purpose of this prop is clear from the name, so I'd just remove the description.

Reply:
👍🏼